src/app/app.module.ts

```diff
@@ -147,6 +148,22 @@ export function configUpdaterFactory(x): any {
   return x[0];
 }
 
+export function getOptionsFactory(arlasAuthService: AuthentificationService): any {
+  const getOptions = () => {
+    const token = arlasAuthService.accessToken !== null ? arlasAuthService.accessToken : null;
+    if (token !== null) {
```

Comment:
I think we should test `undefined` too : `token !== null && token !== undefined`

Reply:
Fixed